py/specter/psf/spotgrid_python_lists.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
```

Comment:
Please remove this file from your branch; I think it is leftover testing code that isn't needed anymore.

Reply:
removed